### DISCUSSION TITLE
[Bug]Landing page currently viewing section tracking not working nomally

### DISCUSSION THREAD

**Discussion by kjsik11** [community/community#12685] [Other Feature Feedback, Questions, & Ideas]
page src: [http://github.com/](http://github.com/)

https://user-images.githubusercontent.com/71022607/157808431-ac0f7867-8021-4d49-b152-9a9934efec85.mov

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
